test/e2e/search_test.go

```diff
@@ -391,4 +391,11 @@ registries = ['{{.Host}}:{{.Port}}']`
 		// cleanup
 		resetRegistriesConfigEnv()
 	})
+
+	// search should fail with nonexist authfile
+	It("podman search fail with nonexist --authfile", func() {
+		search := podmanTest.Podman([]string{"search", "--authfile", "/tmp/nonexist", "alpine"})
+		search.WaitWithDefaultTimeout()
+		Expect(search.ExitCode()).To(Not(Equal(0)))
+	})
```

Comment:
can you add tests for the push, run and play-kube commands?  Are any changes needed for cmd/podman/run.go or cmd/podman/play_kube.go too?  I would suspect so.

Reply:
@QiWang19 can you answer this one please?

Reply:
tests were added

Reply:
I don't see a change to test/e2e/run_test.go, test/e2e/push_test.go or test/e2e/play_kube_test.go in this pr....

Reply:
push tests are in login_logout_test.go. I will add tests to run_test.go

Reply:
:+1:  thx, I'd rather have an additional test in push_test.go, but I won't hold this up further for that.